src/model/task.rs

```diff
@@ -51,7 +52,14 @@ impl Task {
             if verbose > 0 {
                 command.arg("-x");
             }
-            command.current_dir(start_dir).arg("-c");
+
+            if let Some(working_directory) = &self.working_directory {
+                command.current_dir(start_dir.join(working_directory));
```

Comment:
should we check here if `working_directory` is absolute, and in that case, not join with `start_dir`?